pkg/version/check.go

```diff
@@ -135,3 +137,40 @@ func CheckStoreVersion(ctx context.Context, client pd.Client, storeID uint64) er
 	}
 	return nil
 }
+
+// TiCDCClusterVersion is the version of TiCDC cluster
+type TiCDCClusterVersion int
+
+// ticdc cluster version
+const (
+	TiCDCClusterVersionUnknown TiCDCClusterVersion = iota
+	TiCDCClusterVersion4_0
+	TiCDCClusterVersion5_0
```

Comment:
```suggestion
	TiCDCClusterVersionUnknown TiCDCClusterVersion = 0
	TiCDCClusterVersion4_0 TiCDCClusterVersion = 4
	TiCDCClusterVersion5_0 TiCDCClusterVersion = 5
```

Please also add a test to make sure the numerical value of TiCDCClusterVersionX_Y is not changed accidentally.

Reply:
I am afraid that I don't agree with this.
the value of TiCDCClusterVersionX_Y is non-persistent, and the change of TiCDCClusterVersionX_Y will not cause compatibility issues.
Another reason is how to represent the 4.1.X or 5.1.X version if exists in the future.

Reply:
ping @overvenus

Reply:
They are exported, so I don't think we can change their value without considering compatibility. A test can prevent changing their value by accident.